content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Install_the_YubiHSM_Tools_and_Software.adoc

```diff
@@ -0,0 +1,60 @@
+
+
+== Install the YubiHSM 2 Tools and Software
+
+To complete the procedures in this guide, install the YubiHSM 2 tools and software.
+
+[TIP]
+=====
+A generic prompt, *$*, is used in command line examples in this document. Depending on your command line application, your prompt may be different.
+=====
+
+*Step 1* Unzip the downloaded https://developers.yubico.com/YubiHSM2/Releases/[archives of the SDK] containing the YubiHSM libraries and tools and move the contents to an appropriate location.
```

Comment:
Please make this link relative, like: "link:../../Releases"

Reply:
Done